src/fallbackServiceStub.ts

```diff
@@ -171,6 +175,11 @@ export function generateServiceStub(
             ])
               .then(([ok, buffer]: [boolean, Buffer | ArrayBuffer]) => {
                 const response = responseDecoder(rpc, ok, buffer);
+                if (!callback) {
+                  throw new Error(
```

Comment:
Why would there be no callback? Could we provide a more helpful message or tracing to this end? I wouldn't know how to fix this problem.

Reply:
It's only possible as a result of the bug in the generated code - that's why I wrote `Internal error`. It's basically an assertion that the callback must exist at this point. Which message would you prefer? The customer most likely cannot do anything with it, it's most likely our bug.

Reply:
Alternatively, we might leave no message, but that would be one more `TypeError: undefined is not a function` thing which is equally not helpful.

Reply:
After some thoughts and an attempt to actually test this code (as suggested by @danieljbruce), I just removed this `throw`.